pages/docs/reference/functions.md

```diff
@@ -123,52 +128,48 @@ fun reformat(str: String,
 /*...*/
 }
 ```
-</div>
-
-We could call this using default arguments:
-
-<div class="sample" markdown="1" theme="idea" data-highlight-only>
 
-```kotlin
-reformat(str)
-```
 </div>
 
-However, when calling it with non-default, the call would look something like:
+In a function call, you can use names just for some arguments:
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
 
 ```kotlin
-reformat(str, true, true, false, '_')
+reformat(str,
+    true,
+    upperCaseFirstLetter = true,
+    divideByCamelHumps = true,
+    '_'
+)
 ```
+
 </div>
 
-With named arguments we can make the code much more readable:
+You can skip all arguments with default values:
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
 
 ```kotlin
-reformat(str,
-    normalizeCase = true,
-    upperCaseFirstLetter = true,
-    divideByCamelHumps = false,
-    wordSeparator = '_'
-)
+reformat(str)
 ```
+
 </div>
 
-and if we do not need all arguments:
+You can skip some arguments with default values. However, after the first argument that is not in the right place, you need to name all subsequent 
+arguments:
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
 
 ```kotlin
-reformat(str, wordSeparator = '_')
+reformat(str, upperCaseFirstLetter = false, wordSeparator = '_')
 ```
+
 </div>
 
-When a function is called with both positional and named arguments, all the positional arguments should be placed before the first named one. For example, the call `f(1, y = 2)` is allowed, but `f(x = 1, 2)` is not.
 
-[Variable number of arguments (*vararg*{: .keyword })](#variable-number-of-arguments-varargs) can be passed in the named form by using the **spread** operator:
+You can pass a [variable number of arguments (`vararg`)](#variable-number-of-arguments-varargs) with names using the 
+`spread` operator:
```

Comment:
Not sure using a code block for the term *spread* is the best choice here. Maybe change it to italic if we don't like the bold version
```suggestion
*spread* operator:
```

Reply:
We use this formatting .